src/Transition/CreateNewIDFUsingRulesV9_6_0.f90

```diff
@@ -384,6 +384,11 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
 !                 CurArgs = CurArgs + 1
 
               ! If your original object starts with A, insert the rules here
+              CASE('AIRLOOPHVAC:OUTDOORAIRSYSTEM')
+                  CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                  nodiff=.false.
+                  OutArgs(1:3)=InArgs(1:3)
+                  CurArgs = CurArgs - 1
```

Comment:
Simple transition here.  Good.

Reply:
@RKStrand I don't see this rule added to \src\Transition\InputRulesFiles\Rules9-5-0-to-9-6-0.md Since this is merged, you can slip that into one of your other PRs.

Reply:
@mjwitte Whoops!  Totally forgot about that.  I'll attach it to the next defect that I work on.  I'll also update that file because currently it is "blank" but it talks about V9.4 to V9.5 not V9.5 to V9.6.